src/componentLibrary/CTA/CTA.stories.tsx

```diff
@@ -1,6 +1,6 @@
 import { Story, Meta } from '@storybook/react';
 
-import { CTA, CTAProps } from './';
+import { CTA, CTAProps } from '.';
```

Comment:
@peterslany not specifically a comment on this PR, but could you try saving one of your local files to see if Prettier makes this change for you? It doesn't for me. I'm happy with either file path, but I'd like it to be consistent so I want to make sure our editors are making changes in the same way.

Reply:
@tomjeatt 
That sounds good to me. Let me try once more on my side.

Reply:
ok :) I don't want to revert the changes, but @crypto-engineer could you look into what's actually causing them to be changed? e.g. something in your local Prettier or ESLint config. I'd actually quite like to use it across the team, because these kinds of things can get a bit messy so would be nice to have something to tidy it up for us.

Reply:
Thank you @tomjeatt .